### DISCUSSION TITLE
Welcome to wslg Discussions!

### DISCUSSION THREAD

**Discussion by spronovo** [microsoft/wslg#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayeminoosc** [comment]
Hi My name is Aye Min and was exploring wsl & wslg. I was wondering why wslg requrie certain version of window. as per my understanding wlsg is another x server,  as long as linux distro communicate with X protocols, it should be independent of wsl project.

**Reply by Masamune3210** [reply]
As far as I remember, due to how the system has to be implemented it requires a certain version of certain things within Windows. 
Its much much more than just a X server
